.pre-commit-config.yaml

```diff
@@ -49,6 +49,6 @@ repos:
           - dask
           - numpy
           - pytest
-          - tornado
+          - tornado >=6.2
```

Comment:
Changes to this file are exclusively to force purging the cache in github actions and on developers' local machines.
A clean host with no cache and no pinned versions will behave the same.